requirements.txt

```diff
@@ -1,7 +1,7 @@
 --extra-index-url https://pypi.dwavesys.com/simple
 
-dimod==0.9.16
-dwave-cloud-client==0.8.7
+dimod==0.10.6
```

Comment:
Why increment the dimod version in this PR?

Reply:
just a bump to the latest dimod. unrelated to this pr